crates/loro-internal/src/kv_store.rs

```diff
@@ -110,110 +117,1619 @@ mod default_binary_format {
     }
 }
 
+mod sst_binary_format {
```

Comment:
This mod is too long to understand the encoding schema without a doc. 
Please include the doc here